server.js

```diff
@@ -1,30 +1,97 @@
 import express from "express";
 import cors from "cors";
+import netflixData from "./data/netflix-titles.json";
+import listEndpoints from "express-list-endpoints";
```

Comment:
This was nice! but in the browser ( https://project-express-api-w5otwao4va-lz.a.run.app/endpoints ) the last two does not lead anywhere, but its a nice touch anyway!